pkg/cmd/issue/shared/lookup.go

```diff
@@ -41,6 +36,70 @@ func IssueFromArgWithFields(httpClient *http.Client, baseRepoFn func() (ghrepo.I
 	return issue, baseRepo, err
 }
 
+// IssuesFromArgWithFields loads 1 or more issues or pull requests with the specified fields. If some of the fields
+// could not be fetched by GraphQL, this returns non-nil issues and a *PartialLoadError.
+func IssuesFromArgsWithFields(httpClient *http.Client, baseRepoFn func() (ghrepo.Interface, error), args []string, fields []string) ([]*api.Issue, ghrepo.Interface, error) {
```

Comment:
Due to the async nature of this function's implementation, the resulting slice of `api.Issue` results does not necessarily match the order of input `args`. It doesn't seem like that negatively affects the rest of the `edit` implementation, but I wonder will this be apparent to future callers of this function. Just a thought!